src/phylum/ci/ci_base.py

```diff
@@ -449,10 +449,15 @@ def _get_project_threshold_info(self, project_thresholds: dict, threshold_type:
         if threshold is None:
             risk_domain: RiskDomain = PROJECT_THRESHOLD_OPTIONS.get(threshold_type, RiskDomain("", "", ""))
             threshold = project_thresholds.get(risk_domain.project_name)
-            req_src = "project setting"
+            req_src = "project per-axis threshold"
             if threshold is None:
                 threshold = 1.0
                 req_src = "N/A (fail safe)"
+
+            total_threshold = project_thresholds.get("total", 0.0)
```

Comment:
```suggestion
            total_threshold = project_thresholds.get("total", 1.0)
```
If the `total` project threshold key is not available in the json analysis results, shouldn't the default be the most secure value instead of the least secure value?

Reply:
I can see that argument. But I think it makes more sense to reserve the `1.0` fail safe for the case when neither threshold is present. If the per-axis threshold exists and the total threshold is missing, we might as well just use the per-axis threshold.